test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -144,6 +144,80 @@ describe('v1.0.0 to v2.0.0', function () {
                 done();
             });
         });
+
+        it('should override auth with legacy attributes if they exist', function (done) {
+            var options = {
+                    inputVersion: '1.0.0',
+                    outputVersion: '2.0.0',
+                    retainIds: true
+                },
+                source = {
+                    currentHelper: 'basicAuth',
+                    helperAttributes: {
+                        id: 'basic',
+                        username: 'username',
+                        password: 'password'
+                    },
+                    auth: {
+                        type: 'noauth'
+                    }
+                };
+
+            transformer.convertSingle(source, options, function (err, converted) {
+                expect(err).to.not.be.ok;
+
+                // remove `undefined` properties for testing
+                converted = JSON.parse(JSON.stringify(converted));
+
+                expect(converted.request.auth).to.eql({
+                    type: 'basic',
+                    basic: {
+                        username: 'username',
+                        password: 'password',
+                        showPassword: false
```

Comment:
where is the `showPassword` coming from?

Reply:
This is an inbuilt transformer default. Removing it will break other transformations.